guides/environments.md

```diff
@@ -73,6 +73,17 @@ Note that your `xdn.config.js` file is loaded during deployment to configure the
 deploy your site, there won't be any environment variables defined, so you need to include defaults in `xdn.config.js` as
 shown in the example above.
 
+### Permitted Environment Variables
```

Comment:
There already is a section on environment variables:
https://developer.moovweb.com/guides/environments#section_environment_variables

I don't understand what you're trying say with this section.

Reply:
A user can choose any name for an environment variable as long as it doesn't start with xdn_